public/app/core/components/Select/FolderPicker.tsx

```diff
@@ -96,6 +100,12 @@ export function FolderPicker(props: Props) {
       const resultsAfterMapAndFilter = mapSearchHitsToOptions(searchHits, filter);
       const options: Array<SelectableValue<string>> = resultsAfterMapAndFilter;
 
+      reportInteraction('folder_picker_search_results_loaded', {
```

Comment:
Also, now that I think if it, We have RS guidelines for naming: https://docs.google.com/document/d/1MTS5O5qsxtDptzN6AvdMRRGP38pvFc1NULsQKQN_z2U/edit#bookmark=id.z50v0471q8l7 

So something like:
* `grafana_folder_picker_search_results_loaded`
* `grafana_folder_picker_folder_created` or `grafana_folder_picker_create_folder_clicked`

Might be more appropriate?